### ISSUE TITLE
Overfitting in a purely offline setting

### ISSUE THREAD

**Issue by adithyamurali** [ColinQiyangLi/qc#10]
Hi @ColinQiyangLi 

Awesome work and thanks for releasing the code! I'm trying to apply Q-chunking to a purely offline setting. I have 50 positive trajectories and 50 negative demos where the task did not succeeded. I just sequentially walk through these trajectories to create the dataset for training (around 7K transitions for training and validation)

I applied Equation 7 (in the paper) to this small dataset. I'm noticing that on the training set, the loss is decreasing properly. However, the loss is rising in the validation set (10% of data) - the model is clearly overfitting.

Do you have any suggestions on mitigating this? Is the solution to collect more data or finetune this Q function in offline-to-online online rollouts?

thanks!

**Comment by ColinQiyangLi** [comment]
Hi @adithyamurali 

Thanks for your interest in our work! Some questions to help me understand it a bit better:

(1) Do you know which model (policy or critic) overfits more (e.g., is the validation flow-matching loss also increasing?)
(2) How well does training a policy (e.g., flow, action chunking) on the positive demos do?
(3) Which variant of the algorithm did you use (e.g., the best-of-N variant of the FQL variant?)

My best guest based on what you said so far would be to either (1) reduce the model size, (2) find ways to do data augmentations (e.g., image augmentation if you are operating in the pixel domain), or as you mentioned (3) collect more data/online fine-tuning, which should fix these pathological overfitting issues

**Comment by ColinQiyangLi** [comment]
Closing this for now due to inactivity but feel free to re-open it if you have more questions!
